service/controller/v13/encrypter/spec.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/certs/legacy"
```

Comment:
I noticed this when we worked on the k8s libs updates as well. We still use some legacy crap here. Would be cool to work towards cleaning this up a bit in the future.

Reply:
AIUI we can't have both the new certs and the legacy package vendored.

The legacy certs are used in the v1 (legacy) resource. We didn't want to invest time unpicking it there. So this is waiting on the cluster upgrades until we can delete v1.

It sucks but I've tried upgrading previously and got blocked by this.

Reply:
I see. Makes sense.